Makefile

```diff
@@ -5,6 +5,9 @@ IMG_EPSS=$(foreach dotfile,$(IMG_SRCS),build/img/$(patsubst %.dot,%.eps,$(lastwo
 
 images: $(IMG_EPSS)
 
+build/img:
+	mkdir -p build/img
```

Comment:
Try upgrading `make`. I don't know why, but 3.81 can't handle this, but a newer version, e.g., 4.3 is fine. This is supposed to be handled by the order only prerequisite declared on the `build/img/%.eps` target. The target `build/img/` should match is `%/`, but 3.81 can't figure it out for some reason.

Reply:
Interesting, it's strange that Make 3.81 doesn't handle it correctly. I've reverted the change.